src/MapperInterface.php

```diff
@@ -39,6 +39,9 @@ public function load(object $from, string $toClass, array $context): object;
      *
      * Receives the "to object" returned from load().
      *
+     * @template TFrom of object
```

Comment:
TFrom could be removed entirely. As it is only used in the param which is the source for inferring it, it doesn't provide any benefit over the object type

Reply:
done. removed `TFrom` everywhere